File: common/src/main/kotlin/generations/gg/rarecandylib/common/client/util/MinecraftClientGameProvider.java
package generations.gg.rarecandylib.common.client.util;

public class MinecraftClientGameProvider {
    private static final double START_TIME = System.currentTimeMillis();

    public static double getTimePassed() {
        return (System.currentTimeMillis() - START_TIME) / 1000f;
    }
}

File: common/src/main/kotlin/generations/gg/rarecandylib/common/client/render/rarecandy/MinecraftClientGameProvider.kt
package generations.gg.rarecandylib.common.client.render.rarecandy

object MinecraftClientGameProvider {
    private val START_TIME = System.currentTimeMillis().toDouble()

    val timePassed: Double
        get() = (System.currentTimeMillis() - START_TIME) / 1000f
}